acknowledgments receiver is data sender (#265)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

  critical for high bandwidth connections.

- Similarly, receiving and processing UDP packets can also be CPU intensive, and

>   reducing number of packets that only contain acknowledgments also reduces
>   this cost at a acknowledgments receiver which is the data sender.

Reviewer: ```suggestion
  reducing the number of acknowledgments also reduces the processing
  cost on the sender side.
```

Author: I thinks this change mostly reverts my proposed change. I think the original sentence was correct, so if people think more words don't help, we don't have to accept this PR. But I thought it might clarify things. I guess it would be good if @LPardue takes a look who raised the issue.

Reviewer 2: So after thinking some more, IMO there's a bit too much passive voice in the current document, making it hard to determine the subject and the effects. I made a new PR that makes things clearer to me at least.
